tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
Is this still needed? And likewise for `skipLibCheck`

Reply:
Actually, for `skipLibCheck`, if I take that away, then when I run `yarn build`, I get errors like:

```
node_modules/eth-json-rpc-middleware/dist/block-cache.d.ts:1:37 - error TS2307: Cannot find module 'eth-block-tracker' or its corresponding type declarations.

1 import { PollingBlockTracker } from 'eth-block-tracker';
                                      ~~~~~~~~~~~~~~~~~~~

node_modules/eth-json-rpc-middleware/dist/block-ref-rewrite.d.ts:1:37 - error TS2307: Cannot find module 'eth-block-tracker' or its corresponding type declarations.

1 import { PollingBlockTracker } from 'eth-block-tracker';
                                      ~~~~~~~~~~~~~~~~~~~

node_modules/eth-json-rpc-middleware/dist/block-ref.d.ts:1:37 - error TS2307: Cannot find module 'eth-block-tracker' or its corresponding type declarations.

1 import { PollingBlockTracker } from 'eth-block-tracker';
                                      ~~~~~~~~~~~~~~~~~~~
```

Looking at the dependencies for `eth-json-rpc-middleware`, it doesn't seem that `eth-block-tracker` is listed there. It's listed in `devDependencies` instead, but I don't think that's right. So that would be need to be fixed first it seems.

Reply:
PR to fix this: https://github.com/MetaMask/eth-json-rpc-middleware/pull/125

Reply:
Bumped `eth-json-rpc-middleware` to 8.1.0 so build errors should no longer be present.